hokusai/staging-config.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
These were already pushed to the clusters with `hokusai config push staging` / `hokusai config push production`

Likewise I created secrets with `hokusai secrets push staging` / `hokusai secrets push production` - they are not checked in but you can pull them locally with  `hokusai secrets pull staging` / `hokusai secrets pull production`, edit them and push them again